content/chainguard/migration/migrations-overview.md

```diff
@@ -290,9 +290,57 @@ In addition, we have a few migration guides in the form of videos:
 * [PyTorch / CUDA 12](/chainguard/chainguard-images/getting-started/pytorch-cuda/)
 * [Ruby](/chainguard/chainguard-images/getting-started/ruby/)
 
-## Troubleshooting Resources
+## Common Pitfalls
```

Comment:
I think this might need to be merged with the tips section. There's a lot of overlap.

Reply:
fair point, I was rushing to get this off my plate and wasn't thinking of how well it all fit